### ISSUE TITLE
vagrant up fails with winrm

### ISSUE THREAD

**Issue by ajes** [rapid7/metasploitable3#224]
## Issue Description
After building metasploitable3 (no errors) I run 

> vagrant up

It has failed with error regarding winrm (see below).

I have tried to install winrm gem, but it didn't help.

## Host System
* OS: Fedora 27 (fully updated)
* Packer Version: v1.1.3
* Vagrant Version: 2.0.1-1.fc27
* VirtualBox Version: 5.2.6-2.fc27

## Command Output
```
$ vagrant up
/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- winrm (LoadError)
	from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/plugins/communicators/winrm/shell.rb:9:in `block in <top (required)>'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/util/silence_warnings.rb:8:in `silence!'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/plugins/communicators/winrm/shell.rb:8:in `<top (required)>'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/plugins/communicators/winrm/communicator.rb:6:in `require_relative'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/plugins/communicators/winrm/communicator.rb:6:in `<top (required)>'
	from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/plugins/communicators/winrm/plugin.rb:15:in `block in <class:Plugin>'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/registry.rb:27:in `get'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/machine.rb:248:in `communicate'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/machine.rb:140:in `initialize'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/vagrantfile.rb:79:in `new'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/vagrantfile.rb:79:in `machine'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/environment.rb:733:in `machine'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/plugin/v2/command.rb:180:in `block in with_target_vms'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/plugin/v2/command.rb:204:in `block in with_target_vms'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/plugin/v2/command.rb:186:in `each'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/plugin/v2/command.rb:186:in `with_target_vms'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/plugins/commands/up/command.rb:87:in `execute'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/cli.rb:42:in `execute'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/lib/vagrant/environment.rb:308:in `cli'
	from /usr/share/vagrant/gems/gems/vagrant-2.0.1/bin/vagrant:138:in `<top (required)>'
	from /usr/share/vagrant/gems/bin/vagrant:23:in `load'
	from /usr/share/vagrant/gems/bin/vagrant:23:in `<main>'
```
```
$ gem list --local | grep winrm
winrm (2.2.3)
```

**Comment by Chan9390** [comment]
Hey @ajes , please try running vagrant after installing the winrm plugins - `vagrant plugin install winrm winrm-fs`

**Comment by ajes** [comment]
Yep. It helped.

**Comment by keltik85** [comment]
I have the same problem, but when I run the command `vagrant plugin install winrm winrm-fs`
I get these errors:

```
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

conflicting dependencies fog-core (~> 1.43.0) and fog-core (= 1.45.0)
  Activated fog-core-1.45.0
  which does not match conflicting dependency (~> 1.43.0)

  Conflicting dependency chains:
    fog-core (= 1.45.0), 1.45.0 activated

  versus:
    vagrant-libvirt (> 0), 0.0.41 activated, depends on
    fog-core (~> 1.43.0)
```

and 

```
Bundler, the underlying system Vagrant uses to install plugins,
reported an error. The error is shown below. These errors are usually
caused by misconfigured plugin installations or transient network
issues. The error from Bundler is:

conflicting dependencies fog-core (~> 1.43.0) and fog-core (= 1.45.0)
  Activated fog-core-1.45.0
  which does not match conflicting dependency (~> 1.43.0)

  Conflicting dependency chains:
    fog-core (= 1.45.0), 1.45.0 activated

  versus:
    vagrant-libvirt (> 0), 0.0.41 activated, depends on
    fog-core (~> 1.43.0)

```

What to do?

**Comment by dmitvitalii** [comment]
The issue still persists.

## Host System
* OS: Fedora 29
* Vagrant Version: 2.1.2

## Command Output
```
Traceback (most recent call last):
	25: from /usr/share/vagrant/gems/bin/vagrant:23:in `<main>'
	24: from /usr/share/vagrant/gems/bin/vagrant:23:in `load'
	23: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/bin/vagrant:161:in `<top (required)>'
	22: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/environment.rb:275:in `cli'
	21: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/cli.rb:54:in `execute'
	20: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/plugins/commands/up/command.rb:87:in `execute'
	19: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/plugin/v2/command.rb:186:in `with_target_vms'
	18: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/plugin/v2/command.rb:186:in `each'
	17: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/plugin/v2/command.rb:204:in `block in with_target_vms'
	16: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/plugin/v2/command.rb:180:in `block in with_target_vms'
	15: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/environment.rb:700:in `machine'
	14: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/vagrantfile.rb:79:in `machine'
	13: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/vagrantfile.rb:79:in `new'
	12: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/machine.rb:141:in `initialize'
	11: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/machine.rb:260:in `communicate'
	10: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/registry.rb:27:in `get'
	 9: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/plugins/communicators/winrm/plugin.rb:15:in `block in <class:Plugin>'
	 8: from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:59:in `require'
	 7: from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:59:in `require'
	 6: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/plugins/communicators/winrm/communicator.rb:6:in `<top (required)>'
	 5: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/plugins/communicators/winrm/communicator.rb:6:in `require_relative'
	 4: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/plugins/communicators/winrm/shell.rb:8:in `<top (required)>'
	 3: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/lib/vagrant/util/silence_warnings.rb:8:in `silence!'
	 2: from /usr/share/vagrant/gems/gems/vagrant-2.1.2/plugins/communicators/winrm/shell.rb:9:in `block in <top (required)>'
	 1: from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:59:in `require'
/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:59:in `require': cannot load such file -- winrm (LoadError)
```

`$ vagrant plugin install winrm winrm-fs`  didn't help.

**Comment by ryancraig** [comment]
Although this issue has been closed, here's the solution that worked for me...

I am using Fedora 30. `vagrant-2.2.4-2.fc30.noarch` was installed and I ran into the `require': cannot load such file -- winrm (LoadError)` error.

I removed `vagrant-2.2.4-2.fc30.noarch` and installed the CentOS 64bit binary available here https://www.vagrantup.com/downloads.html. I then installed the downloaded rpm. Everything now works when I vagrant up a Windows box on my Fedora 30 workstation. 

In conclusion, you should NOT attempt to install the winrm plugin.  WinRM is now delivered in the Vagrant core binary. Remove the old Vagrant package and install the current release provided by HashiCorp.

**Comment by ryancraig** [comment]
Although this issue has been closed, here's the solution that worked for me...

I am using Fedora 30. `vagrant-2.2.4-2.fc30.noarch` was installed and I ran into the `require': cannot load such file -- winrm (LoadError)` error.

I removed `vagrant-2.2.4-2.fc30.noarch` and installed the CentOS 64bit binary available here https://www.vagrantup.com/downloads.html. I then installed the downloaded rpm. Everything now works when I vagrant up a Windows box on my Fedora 30 workstation. 

In conclusion, you should NOT attempt to install the winrm plugin.  WinRM is now delivered in the Vagrant core binary. Remove the old Vagrant package and install the current release provided by HashiCorp.
